packages/components/radio-group/types/utils/helper-functions.d.ts

```diff
@@ -0,0 +1,2 @@
+export function getDirectionByKey(keyCode: any): string;
+export function getNextElementIndex(direction: any, index: any, collectionLength: any): any;
```

Comment:
Again, don't know if we have to export these since they are used only internally.